brands/shop/supermarket.json

```
@@ -2617,6 +2617,31 @@
       "shop": "supermarket"
     }
   },
+  "shop/supermarket|MEGAドン・キホーテ": {
+    "countryCodes": ["jp"],
+    "matchNames": ["メガ・ドン・キホーテ"],
+    "matchTags": [
+      "amenity/general",
+      "amenity/retail",
+      "amenity/variety_store"
+    ],
+    "nomatch": ["amenity/variety_store|ドン・キホーテ"],
+    "tags": {
+      "brand": "ドン・キホーテ",
+      "brand:en": "Don Quijote",
+      "brand:ja": "ドン・キホーテ",
+      "brand:wikidata": "Q1185381",
+      "brand:wikipedia": "ja:ドン・キホーテ (企業)",
+      "name": "MEGAドン・キホーテ",
+      "name:en": "MEGA Don Quijote",
+      "name:ja": "MEGAドン・キホーテ",
+      "opening_hours": "24/7",
+      "shop": "general",
```

Should be like this so it matches the `shop/supermarket` above..
```suggestion
      "shop": "supermarket",
```

---

Fixed